scripts/build-deps

```diff
@@ -40,18 +40,16 @@ fi
 cd $PYAV_LIBRARY
 
 echo ./configure
-if [[ "$PYAV_LIBRARY_NAME" == ffmpeg ]]; then
-    CONFFLAGS=--disable-stripping
-fi
 ./configure \
-    --disable-static \
-    --enable-shared \
     --disable-doc \
-    --disable-optimizations \
     --disable-mmx \
+    --disable-optimizations \
+    --disable-static \
+    --disable-stripping \
     --enable-debug=3 \
     --enable-gpl \
     --enable-libx264 \
+    --enable-shared \
     $CONFFLAGS \
```

Comment:
Does $CONFFLAGS do anything? If not, either remove it or call it `PYAV_LIBRARY_CONFIG` or something. (Better to just remove since this is for testing.)

Reply:
That came off rather commanding. Sorry. Was just writing notes, not trying to issue orders.

Reply:
No worries, the first comment was "I love it" ;)